docs/spinner.md

```diff
@@ -2,19 +2,28 @@
 title: Spinner
 ---
 
+[cli-ux](https://github.com/oclif/cli-ux) provides a simple `cli.action`, for more complex progress indicators we recommend using the [listr](https://www.npmjs.com/package/listr) library.
+
 ## `cli-ux.action`
 
 Shows a basic spinner
 
 ```typescript
-// start the spinner
-cli.action.start('starting a process')
-// show on stdout instead of stderr
-cli.action.start('starting a process', 'initializing', {stdout: true})
-
-// stop the spinner
-cli.action.stop() // shows 'starting a process... done'
-cli.action.stop('custom message') // shows 'starting a process... custom message'
+import {Command} from '@oclif/command'
+import cli from 'cli-ux'
+
+export class MyCommand extends Command {
+  async run() {
+    // start the spinner
+    cli.action.start('starting a process')
+    // show on stdout instead of stderr
+    cli.action.start('starting a process', 'initializing', {stdout: true})
+
+    // stop the spinner
+    cli.action.stop() // shows 'starting a process... done'
+    cli.action.stop('custom message') // shows 'starting a process... custom message'
+  }
+}
```

Comment:
```suggestion
export class MyCommand extends Command {
  async run() {
    // start the spinner
    cli.action.start('starting a process')
    // do some action...
    // stop the spinner
    cli.action.stop() // shows 'starting a process... done'
    
    // show on stdout instead of stderr
    cli.action.start('starting a process', 'initializing', {stdout: true})    
    // do some action...
    // stop the spinner with a custom message
    cli.action.stop('custom message') // shows 'starting a process... custom message'
  }
}
```